### DISCUSSION TITLE
metadata original_filename

### DISCUSSION THREAD

**Discussion by pgswow** [pocketbase/pocketbase#1280] [Q&A] [answered]
Hello! In 0.9 release you added: The original uploaded file name is now stored as metadata under the original_filename key.
Can i accsess to this key from js_api ?

**Comment by ganigeorgiev** [comment, marked as answer]
No. This is available only if you extend PocketBase and when accessing the file via the internal filesystem abstraction, eg:

```go
fs, err := app.NewFilesystem()
if err != nil {
	return err
}
defer fs.Close()

attrs, err := fs.Attributes(fileKey)
if err != nil {
	return err
}

log.Println(attrs.Metadata["original_filename"])
```
